frontend/src/components/Header.vue

```diff
@@ -1,69 +1,152 @@
 <template>
-  <div class="header p-3">
-    <Language />
-    <h1><a href="https://www.pah.org.pl/">{{ $t("header.polish_humanitarian_action") }}</a></h1>
-    <b-nav
-      fill
-    >
-      <b-nav-item
-        v-for="link in links"
-        :to="link.to"
-        :key="link.text"
+  <div class="header">
+    <a
+      class="logo-container"
+      href="https://www.pah.org.pl/"
+      target="_blank">
+      <img
+        class="logo"
+        :src="logo" >
+    </a>
+    <nav class="menu">
+      <b-nav
+        fill
       >
-        {{ link.text }}
-      </b-nav-item>
-      <LoginStatus v-bind="user"/>
-    </b-nav>
+        <b-nav-item
```

Comment:
Can menu be a separate component? Now it's duplicated in Header and App components.